docs/labs/docker-plugin.rst

```diff
@@ -88,17 +88,17 @@ Upload these files to ``/etc/flocker/plugin.key`` and ``/etc/flocker/plugin.crt`
 
 Then perform the following instructions on each of the hosts where you want to install the Flocker Docker plugin.
 
-Install Experimental Docker
----------------------------
+Install Docker
+--------------
 
-Install the experimental build of Docker:
+Install Docker 1.8 or later:
 
 .. prompt:: bash $
 
-    wget -qO- https://experimental.docker.com/ | sudo sh
+    wget -qO- https://get.docker.com/ | sudo sh
```

Comment:
:question: This is a dangerous pattern, see https://www.seancassidy.me/dont-pipe-to-your-shell.html.